CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.14.6 (unreleased)
+
+Documentation:
+
+  - Fix links about `:source` in `Gemfile` man page
```

Comment:
Ah, thanks for clarifying. Saw a mention of it in [the pull request guidelines](https://github.com/bundler/bundler/blob/master/doc/development/PULL_REQUESTS.md#changelogmd), so I went ahead and added it.

Should I remove that from this branch? And if so, would you prefer for me to do that in another commit or to amend my existing commit?